app/views/public/schedule/style.css.erb

```diff
@@ -103,5 +103,5 @@
 /* Custom Conference CSS */
 
 <% if @conference.schedule_custom_css.present? %>
-  <%= @conference.schedule_custom_css.gsub(/\{([^}]*)\}/m){ '{' + sanitize_css($1) + '}'; } %>
```

Comment:
Why? This field is only writable by admins, which have numerous other ways of supplying malicious content. Also, as I wrote in my commit logs, the naive cleanup method used in this case messes with more complex CSS.